config.sh

```diff
@@ -266,7 +266,7 @@ WIN_PERL=${WIN_PERL:-"C:/perl/bin/perl.exe"}
 # Android default settings and tools
 if test "${OS}" = "android" ; then
     ANDROID_NDK_VERSION=${ANDROID_NDK_VERSION:-r10d}
-    ANDROID_PLATFORM=${ANDROID_PLATFORM:-android-8}
+    ANDROID_PLATFORM=${ANDROID_PLATFORM:-android-10}
```

Comment:
Although we don't have very many (any?) users of Android 2.2, a release note would be useful to say that we now require 2.3.3.